test/integration/scheduler_perf/README.md

```diff
@@ -68,6 +68,25 @@ To produce a cpu profile:
 make test-integration WHAT=./test/integration/scheduler_perf KUBE_TIMEOUT="-timeout=3600s" KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run=^$$ -benchtime=1ns -bench=BenchmarkPerfScheduling -cpuprofile ~/cpu-profile.out"
 ```
 
+### How to configure bechmark tests
+
+Configuration file located under config/performance-config.yaml contains a list of templates.
+Each template allows to set:
+- node manifest
+- manifests for initial and testing pod
+- number of nodes, number of initial and testing pods
+- templates for PVs and PVCs
+- feature gates
+
+See `simpleTestCases` data type implementation for available configuration for each template.
+
+Initial pods create a state of a cluster before the scheduler performance measurement can begin.
+Testing pods are then subject to performance measurement.
+
+The configuration file under config/performance-config.yaml contains a default list of templates to cover
+various scenarios. In case you want to add your own, you can extend the list with new templates.
+It's also possible to extend `simpleTestCases`, resp. its underlying data types to enrich possible test cases.
```

Comment:
I don't understand this sentence, what is "resp."?

Reply:
resp. = respectively

Reply:
I rephrased the sentence.